tests/base_cluster.py

```diff
@@ -117,7 +117,7 @@ def exec_cmd_on_host(self, host, cmd, user=None, raise_error=True,
         pass
 
     @abc.abstractmethod
-    def run_script_on_host(self, script_contents, host):
+    def run_script_on_host(self, script_contents, host, tty=True):
```

Comment:
Can you please braindump your tty knowledge here so we capture it for posterity?